src/Core/src/Platform/MauiContext.Android.cs

```diff
@@ -13,7 +16,7 @@ public MauiContext(IServiceProvider services, Context context)
 			_context = new WeakReference<Context>(context ?? throw new ArgumentNullException(nameof(context)));
 		}
 
-		public MauiContext(Context context)
+		internal MauiContext(Context context)
```

Comment:
So now users can't create a MauiContext ?! they have to create a ScopedMauiContext ?